### PR TITLE
feat(tocco-ui): show unit in DurationEdit if not disturbing

### PR DESCRIPTION
**Pull request by niklaushug**
- Label of StatedValue covers units of DurationEdit which looks ugly and is confusing.
- Show units only if focus is on DurationEdit or if hours or minutes are set.

### CODE REVIEW

File: packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ class DurationEdit extends React.Component {
     this.hoursShadow = React.createRef()
     this.minutesShadow = React.createRef()
     this.state = {
-      ...this.millisecondsToDuration(props.value)
+      ...this.millisecondsToDuration(props.value),
+      focus: props.value >= 0
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js]: focus is true when there is a value >= 0
That sounds confusing. Focus should be true when the field is focused. Maybe call the state `showUnits`?

File: packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js [hunk 2]
```diff
@@ -95,6 +96,9 @@ class DurationEdit extends React.Component {
     this.props.onChange(calculateMilliseconds(hoursValue, minutesValue))
   }
 
+  handleOnBlur = () => this.setState({focus: (this.state.hours >= 1 || this.state.minutes >= 1)})
```
Comment (khanlifts) [inline packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js]: 0 0 
This display without labels can be confusing. Are you sure you want to hide the labels for this case?

Comment (MethodenMann) [reply packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js]: cyril is right. 0 is a valid value.

Comment (niklaushug) [reply packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js]: state.hours and state.minutes can be integer or string. Hence I prefer `this.setState({focus: (this.state.hours.length >= 1 || this.state.minutes.length >= 1)})` over `this.setState({focus: (parseInt(this.state.hours) >= 0 || parseInt(this.state.minutes) >= 0)})`.

File: packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js [hunk 3]
```diff
@@ -95,6 +96,9 @@ class DurationEdit extends React.Component {
     this.props.onChange(calculateMilliseconds(hoursValue, minutesValue))
   }
 
+  handleOnBlur = () => this.setState({showUnits: (this.state.hours.length >= 1 || this.state.minutes.length >= 1)})
```
Comment (khanlifts) [inline packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js]: When I execute the following steps only a `1` is shown:
1. Load the component.
2. Delete minutes. (Or hours respectively.)
3. Click outside.
I think there should be `1 hrs 0 min`.

**Pull request merged**